protected/humhub/modules/user/views/account/editSettings.php

```diff
@@ -29,15 +29,17 @@
     ]); ?>
 <?php endif; ?>
 
-<?php if ($isEnabledOnlineStatus) : ?>
+<?php if ($isEnabledOnlineStatus): ?>
     <?= $form->field($model, 'hideOnlineStatus')->checkbox(); ?>
 <?php endif; ?>
 
-<?php if (Yii::$app->getModule('tour')->settings->get('enable') == 1) : ?>
+<?php if (Yii::$app->getModule('tour')->settings->get('enable') == 1): ?>
     <?= $form->field($model, 'show_introduction_tour')->checkbox(); ?>
 <?php endif; ?>
 
-<?php if (Yii::$app->getModule('user')->allowBlockUsers()) : ?>
+<?= $form->field($model, 'markdownEditorMode')->radioList($model->getEditorModeList()) ?>
```

Comment:
I'm not much of a fan of `radioList`, `dropDownList` would be much cleaner here. :thinking:

Reply:
@luke- Should I change `radioList` to `dropDownList` or not?

Reply:
@yuriimaz Yes please, it's a good suggestion.

Reply:
Done

Reply:
Beautifully done!